### DISCUSSION TITLE
LinkFree infrastructure discussion

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/LinkFree#6504] [Show and tell]
Notes from Community call, leave your thoughts in threads below 👇 

### Atlas

- Security: Lock down Atlas to only Vercel app
   - use `pem` as part of the connection?
   - VPC? https://www.mongodb.com/docs/atlas/reference/faq/connection-changes/

### Vercel

- [x] redirect linkfree.eddiejaoude.io to linkfree.io in Vercel
- [ ] enable preview of pull request on Vercel (exclude profiles `data/`)
- [ ] Changelog twitter account https://twitter.com/vercel_changes/status/1651966112328826886?s=20

### NextJS

- SEO
- Performance

### LinkFree App

- Manual changelog to have author and link to Pull Request

### Communication

- [ ] LinkFree OPs channel for outages similar to real world project / company
- [ ] We are using https://upptime.js.org/ (integrate into OPs channel ☝️ )

### Questions

- Vercel course walk through? https://vercel.com/guides
- Vercel usage is high for images and web analytics?

**Comment by loftwah** [comment]
Tailscale is the commercial version of Wireguard. ZeroTier is another similar solution.

On AWS I like having the network set up this way. Hopefully this can be an option we have at Vercel.

https://docs.aws.amazon.com/vpc/latest/userguide/vpc-example-private-subnets-nat.html

All of this can be kept behind a VPN.

**Comment by CBID2** [comment]
Does anyone have any recommended sources on British English grammar? I’m working on the style guide for working on the docs in Linkfree.

**Reply by loftwah** [reply]
British English is the same as Australian English, I think. I use Fowler's Modern English Usage by Henry W. Fowler and R.W. Burchfield: This classic guide to British English usage has been updated and revised to reflect modern language conventions.

**Reply by CBID2** [reply]
Thanks @loftwah!

**Reply by CBID2** [reply]
@loftwah, I found this one: https://www.unr.edu/writing-speaking-center/student-resources/writing-speaking-resources/british-american-english

**Reply by iammihirsig** [reply]
Great Resource

**Comment by loftwah** [comment]
We should set up an ops channel in the Discord server and treat it like a real channel would be treated in a real business. We could integrate all of our services with it, and it would give members in the community a way to see how things would be done in the real world.

**Comment by CBID2** [comment]
As far as the logo change, why not make Flubber be the logo for Linkfree?

**Reply by iammihirsig** [reply]
I guess maybe something more relevant is required

**Comment by timDeHof** [comment]
For images, If we convert the images to a .webp file format. Would that make the website faster? I think lighthouse would suggest it on websites.

**Reply by eddiejaoude** [reply]
`webp` is great, but doesn't work on Safari on Mac so I never use it

**Reply by amandamartin-dev** [reply]
> `webp` is great, but doesn't work on Safari on Mac so I never use it

That's not true anymore - depends on what version of safari, as of 16 it fully supports webp and was partial before that but I do not know percentage of use by browser version where safari is concerned so depends on how far back you want to be able to support with LF.

**Reply by loftwah** [reply]
Whether to use WebP format on a website depends on the number and type of images and performance. JPEG or PNG are ok for a website with only a few photos. LinkFree focuses more on the speed and reliability of the server and platform rather than the image format. I don't know a lot about WebP, though. It might have changed.

**Reply by amandamartin-dev** [reply]
Yeah, I don't know that it would make a difference here truly. Agreed Dean

**Reply by loftwah** [reply]
Hey, I could have just been very confidently wrong :D

**Reply by amandamartin-dev** [reply]
Lmao, no I don't think we have enough images and big enough files truly. But always worth a look and a quick perf test can tell you. I could also likely be wrong 😂

**Comment by CBID2** [comment]
Could there be a section where people post about their experiences with Linkfree (e.g., how to use Linkfree in your job search). Something to promote the product more?

**Reply by eddiejaoude** [reply]
We do have the docs page where people can add their content on LinkFree https://linkfree.io/docs/community-resources

I want to make this more obvious though, maybe a section on the homepage?

**Comment by iammihirsig** [comment]
Adding a footer improve user experience for easy navigation to things like documentations , pages , etc.

**Comment by naomi-lgbt** [comment]
If Vercel provides a static IP for your app, you could restrict the Atlas firewall settings to just that app.

**Reply by loftwah** [reply]
I don't believe it will. Vercel offers automatic scaling, which means it is probably attached to a load balancer, and all you'll get is a FQDN. I could be wrong, and I'd be keen to help look into this if anyone wants to join.

**Reply by eddiejaoude** [reply]
They are announcing new features this week, so they might add it 🤞 

Today they added file storage and databases 🎉
